packages/at_client/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 3.0.54
+- fix: encryptedSharedKeySyncStatusCacheMap being maintained but not used
```

Comment:
"fix: encryptedSharedKeySyncStatusCacheMap being maintained but not used"  ... this is implementation detail ... what I really want to know is, what BUG is being fixed